dependency_support/verilator/verilator.bzl

```diff
@@ -0,0 +1,53 @@
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+
+def _verilator_repository_impl(ctx):
+    ctx.download_and_extract(
+        url = ["https://github.com/verilator/verilator/archive/refs/tags/v4.224.tar.gz"],
+        sha256 = "010ff2b5c76d4dbc2ed4a3278a5599ba35c8ed4c05690e57296d6b281591367b",
+        stripPrefix = "verilator-4.224",
+    )
+
+    ctx.file("WORKSPACE", "workspace(name = {name})\n".format(name = repr(ctx.name)))
+    ctx.symlink(ctx.attr._buildfile, "BUILD")
+
+    # Generate files usually produced / modified by autotools.
+    replace = {
+        "#define PACKAGE_STRING \"\"": "#define PACKAGE_STRING \"Verilator v4.224\"",
+    }
+    ctx.template("src/config_build.h", "src/config_build.h.in", replace, executable = False)
+
+    # TODO(stridge-crumxl) Find a better solution for this.
```

Comment:
This is a horrible ugly hack hardcoding the bison version. Was hoping to get help with this?